stl/inc/xmemory

```diff
@@ -1908,6 +1910,7 @@ _NoThrowFwdIt _Uninitialized_value_construct_n_unchecked1(_NoThrowFwdIt _UFirst,
 }
 #endif // _HAS_IF_CONSTEXPR
 
+#if !_HAS_CXX20

```

Comment:
I believe that the temporary buffer machinery should be controlled by coarse/fine-grained escape hatches.

Reply:
Done with `_HAS_DEPRECATED_TEMPORARY_BUFFER`